recipes/eglibc/eglibc_2.18-r24943.oe

```diff
@@ -3,9 +3,18 @@ require ${PN}.inc
 SRC_REV := "${@d.get('PV').split('-',1)[1]}"
 PV := "${@d.get('PV').split('-',1)[0]}"
 
+RECIPE_FLAGS:machine	+= " eglibc_powerpc_32bit_addr"
```

Comment:
You are free to re-use the class flag in the recipe. But don't redefine it as a recipe flag.